### ISSUE TITLE
Text (text, longtext,...) are exported as hex on JSON export

### ISSUE THREAD

**Issue by 8ele8** [phpmyadmin/phpmyadmin#16887]
### Describe the bug

Text (text, longtext,...) are exported as hex on JSON export.
Noticed since phpMyAdmin 5.1.0
This was not the case in previous versions where text are exported as plain text on JSON export.

### To Reproduce

1. Select the database or the table you want to export
2. Go to "Export" and select the JSON format
3. Confirm with "Go"
4. Open and check the resulting JSON file

### Example of expected vs. actual exported JSON file

The `option_value` is a `longtext`

Expected:
```json
...
{"option_id":"3","option_name":"blogname","option_value":"A WordPress Site","autoload":"yes"}
...
```
Actual:
```json
...
{"option_id":"3","option_name":"blogname","option_value":"0x4120576f726450726573732053697465","autoload":"yes"},
...
```

### Server configuration

- Operating system: Ubuntu 20.4 (focal)
- Web server: nginx 1.18.0
- Database version: MariaDB 10.3
- PHP version: 7.4.3
- phpMyAdmin version: 5.1.0

**Comment by williamdes** [comment]
Hi @8ele8  !
I think this is already fixed in 5.1.1-dev

It looks like #16847

Could you download the latest 5.1 version in development [(phpMyAdmin 5.1+snapshot)](https://www.phpmyadmin.net/downloads/#snapshot_5.1+snapshot) to have a try ?

**Comment by 8ele8** [comment]
Just tested with this latest version - yes, you're right, it's fixed in this version.
Thanks!
